experiments/react-native-uifabric/src/components/Persona/Persona.helpers.ts

```diff
@@ -0,0 +1,66 @@
+import { PersonaSize } from '../PersonaCoin';
+import { ITextTokens } from '../../tokens';
+
+const textFontTable: { [key in PersonaSize]: ITextTokens } = {

```

Comment:
why not make `{ [key in PersonaSize]: ITextTokens }` a type, say `IPersonaFontTable`?

Reply:
will do that.